CREDITS.md

```diff
@@ -0,0 +1,30 @@
+# Emanate Maintainers
+
+- [Ellen Marie Dash (@duckinator)](https://github.com/duckinator)
+- [nicoo (@nbraud)](https://github.com/nbraud)
+
+
+# Emanate Contributors
+
+These are folks who aren't maintainers, but have contributed in some way. :)
+
+- [Samuel Lewis (@ritlew)](https://github.com/ritlew)
+
```

Comment:
```suggestion
- [@kayila](https://github.com/kayila)

```

She caused some interesting/useful discussion in #54 (which I should get off my butt and act on).